src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -282,6 +283,9 @@
                                 c:\\buildArtifacts\\restart.txt" \\
             --type windows-restart --restart-timeout 10m --defer
 
+  - name: Add a windows update customizer to an image template in the cli object cache.
+    text: |
+        az image builder customizer add -n mytemplate -g my-group --customizer-name win_update --type windows-update --search-criteria IsInstalled=0 --filters "exclude:$_.Title -like \\'*Preview*\\'" "include:$true" --update-limit 20 --defer
```

Comment:
`$true` will be interpreted by Bash as [variable expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html). Consider escaping it in Bash.